lib/screens/player_screen.dart

```diff
@@ -173,3 +163,107 @@ class _PlayerScreenContent extends StatelessWidget {
     );
   }
 }
+
+enum PlayerHideable {
+  bigPlayButton(14, 14),
+  queueButton(0, 27),
+  progressSlider(0, 14),
+  twoLineTitle(0, 27),
+  codecInfo(0, 20),
+  loopShuffleButtons(96, 0);
```

Comment:
`width` and `height` is a bit generic I think, it isn't really obvious that these sizes are added up to determine the total used-up space. It's also completely separated from the *actual* size specified in the player widgets, not sure how best to solve that issue.